plugins/cost-insights/src/utils/change.ts

```diff
@@ -45,11 +47,29 @@ export function growthOf(ratio: number, amount?: number) {
 export function getComparedChange(
   dailyCost: Cost,
   metricData: MetricData,
+  duration: Duration,
 ): ChangeStatistic {
   const ratio = dailyCost.change.ratio - metricData.change.ratio;
-  const amount = dailyCost.aggregation.slice().sort(aggregationSort)[0].amount;
+  const previousPeriodTotal = getPreviousPeriodTotalCost(dailyCost, duration);
   return {
     ratio: ratio,
-    amount: amount * ratio,
+    amount: previousPeriodTotal * ratio,
   };
 }
+
+export function getPreviousPeriodTotalCost(
+  dailyCost: Cost,
+  duration: Duration,
+): number {
+  const costsByDate = dailyCost.aggregation.slice().sort(aggregationSort);
+  const nextPeriodStart = moment(costsByDate[0].date).add(
```

Comment:
Hmm, perhaps we should pass `lastCompleteBillingDate` from the hook into `getComparedChange` and pass it through to here for date calculation. There's no API guarantee that we don't have sparse data, so relying on `[0]` feels shaky.